src/app/new-courses-page/new-course-form/new-course-form.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Output } from '@angular/core';
+
+import { Course } from '@app/models';
+
+@Component({
+  selector: 'app-new-course-form',
+  templateUrl: './new-course-form.component.html',
+  styleUrls: ['./new-course-form.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
you don't have any inputs here. I don't think it makes sense to use onPush here.
Although, on push not only about inputs, it still better to use it when you have inputs and they change.

Reply:
removed
thanks)